playbooks/besman-scorecard-0.0.1-steps.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+    echo "Running $ASSESSMENT_TOOL_NAME"
+    
+    curl -X 'GET' \
```

Comment:
Does this step, automatically perform a scorecard assessment or it only fetches the results?
- If it only fetches the result and can be automated move this into the prepare function.
This file should contain the steps to help a user to perform the assesssment.